src/fetch-mock.js

```diff
@@ -56,6 +57,16 @@ function mockResponse (url, config) {
 	return Promise.resolve(new Response(body, opts));
 }
 
+function toRequest(url, options) {
+	var request;
+	if (Request.prototype.isPrototypeOf(url) && !options) {
+		request = url;
```

Comment:
ok, I'm not so good at writing error messages, but how about this one: 'you can pass options to the Request constructor, rather than passing along' ?